src/main.py

```diff
@@ -231,7 +231,7 @@ def run(self):
                                     sub.unsticky()
 
                                 print "Submitting game thread..."
-                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game"))
+                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game",self.TEAM_CODE))

```

Comment:
So there's some funky codes used by the mlb (for instance, Chicago Cubs are `chn` - Chicago National - not `chc` - Chicago Cubs), so I'd hesitate to use this variable that's more synced with the mlb than your website.

I'd check out the `getNotes` function in `Editor.py` [here](https://github.com/toddrob99/Baseball-GDT-Bot/blob/6f0b9c5dafdf617379369155a684d9120c46f6c8/src/editor.py#L8) for a better mapping. You can look at other usages of that function to figure out what vars to use.